REVIEW_GUIDELINES.md

```diff
@@ -76,6 +76,45 @@ image:
 * The use of the `default` function should be avoided if possible in favor of defaults in `values.yaml`.
 * It is usually best to not specify defaults for resources and to just provide sensible values that are commented out as a recommendation, especially when resources are rather high. This makes it easier to test charts on small clusters or Minikube. Setting resources should generally be a conscious choice of the user.
 
+## Persistence
+
+* Persistence should be turned off by default
+* PVCs should support specifying an existing claim
+* Use multi-attach PVCs instead of several small pvcs
+
+
+```yaml
+volumeMounts:
+  - mountPath: /opt/sonarqube/conf
+    name: sonarqube
+    subPath: conf
+  - mountPath: /opt/sonarqube/data
+    name: sonarqube
+    subPath: data
+  - mountPath: /opt/sonarqube/extensions
+    name: sonarqube
+    subPath: extensions
+```
+
+* Storage class should not be overriden by default
+* All options should be shown in README.md
+* Example persistence section in values.yaml:
+
+```yaml
+persistence:
+  enabled: false
+  ## If defined, storageClassName: <storageClass>
+  ## If set to "-", storageClassName: "", which disables dynamic provisioning
+  ## If undefined (the default) or set to null, no storageClassName spec is
+  ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+  ##   GKE, AWS & OpenStack)
+  ##
+  # storageClass: "-"
```

Comment:
I now think it'd be better to add an empty default value. Thoughts?

Reply:
I'm fine with either way